dotnet/WipeAppWriteBucket/WipeAppWriteBucket.csproj

```diff
@@ -0,0 +1,23 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <None Remove="Appwrite" />
+    <None Remove=".gitignore" />
+    <None Remove="Newtonsoft.Json" />
+    <None Remove="Models\" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Appwrite" Version="0.2.0" />

```

Comment:
Since the Appwrite .NET SDK is not to be used here (as it is out of date), please remove the `PackageReference`

Reply:
Done